Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -693,7 +693,7 @@
     <!-- YTXT: subtitle for legal information page, contact section -->
     <string name="information_legal_subtitle_contact">"E-mail: CoronaWarnApp@rki.de"</string>
     <!-- YTXT: subtitle for legal information page, open contact form -->
```

Comment:
add that this only has to be "translated" into the english url, other languages aren't available

Reply:
update this comment in the german strings values-de